.github/workflows/coverage.yml

```diff
@@ -16,8 +16,10 @@ jobs:
           go-version: 1.21
       - name: Checkout code
         uses: actions/checkout@v2
+      - name: Set up Just
+        uses: extractions/setup-just@v1
       - name: Generate coverage report
-        run: go test -v  -coverprofile=profile.cov ./...
+        run: just coverage
       - name: Parse overall coverage
         run: coverage=$(go tool cover -func profile.cov | tail -1 | rev | cut -f 1 | rev) && echo "COVERAGE=$coverage" >> $GITHUB_ENV 
```

Comment:
should this be a `just` recipe too?

Reply:
Yes, I switched over all our actions to use `just` recipes where possible so that we only have to change things in the `justfile` when necessary!